Fix RedisJSON doc for POJOs (#3200)
redis/jedis: docs/redisjson.md

}
```

Now we can create some students and store them in Redis as JSON:

```java
final Gson gson = new Gson();

Student maya = new Student("Maya", "Jayavant");
client.jsonSet("student:111", gson.toJson(maya));

Student oliwia = new Student("Oliwia", "Jagoda");
client.jsonSet("student:112", gson.toJson(oliwia));
```

> Some other ways to store POJOs as JSON:

Author: ```suggestion
Some of other ways to store POJOs as JSON:
```

> There are other ways to directly store POJOs as JSON:

Author: ```suggestion
Some other ways to store POJOs as JSON:
```
